### PR TITLE
js-packages: remove momentjs

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

File: meinberlin/apps/plans/assets/PlansList.jsx [hunk 1]
```diff
@@ -130,9 +130,13 @@ class PlansList extends React.Component {
...
-    let newDate = Date.parse(item.future_phase.replace(/ /g, 'T'))
-    newDate = Moment(newDate).format('DD.MM.YYYY')
-    return newDate
+    const date = Date.parse(item.future_phase.replace(/ /g, 'T'))
+    const dateStyle = {
+      year: 'numeric',
+      month: '2-digit',
+      day: '2-digit'
+    }
+    return toLocaleDate(date, undefined, dateStyle)
```
Comment (fuzzylogic2000) [inline meinberlin/apps/plans/assets/PlansList.jsx:139]: Undefined just means that it takes the default of toLocaleDate?

**Review by fuzzylogic2000 — COMMENTED** [review]
I have questions. :) And somehow remember, that we did it differently elsewhere? Is that true?

File: meinberlin/apps/plans/assets/PlansList.jsx [hunk 2]
```diff
@@ -130,9 +130,13 @@ class PlansList extends React.Component {
   }
 
   getDate (item) {
-    let newDate = Date.parse(item.future_phase.replace(/ /g, 'T'))
-    newDate = Moment(newDate).format('DD.MM.YYYY')
-    return newDate
+    const date = Date.parse(item.future_phase.replace(/ /g, 'T'))
```
[the remark this answers is not in the document]
Comment (khamui) [reply meinberlin/apps/plans/assets/PlansList.jsx:133]: I just took it from before tbh. Will check if it still works without replacing.

**Comment by khamui** [comment]
> I have questions. :) And somehow remember, that we did it differently elsewhere? Is that true?

Not sure, the helper function we used for budgeting list items, have been moved to contrib now. And I think we only use this function now for the dates?

File: meinberlin/apps/plans/assets/PlansList.jsx (hunk 1 above)
Comment (goapunk) [reply meinberlin/apps/plans/assets/PlansList.jsx:139]: we could use the browser language but then mb is only in german anyway, so I guess we better stick with that?

File: meinberlin/apps/plans/assets/PlansList.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (khamui) [reply meinberlin/apps/plans/assets/PlansList.jsx]: indeed it is not needed to replace string here 👍

**Review by fuzzylogic2000 — APPROVED** [review]
Nice!

**Pull request merged**